### PR TITLE
6245410: javax.swing.text.html.CSS.Attribute: BACKGROUND_POSITION is not w3c spec compliant

### PR DESCRIPTION
**Pull request by prsadhuk**
[CSS spec for background-position](https://www.w3.org/TR/CSS22/colors.html#propdef-background-position)
mention the initial value should be "0% 0%" but JDK has the value as "null".
Rectified the spec violation. 
No regression is seen in client tests..

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-6245410](https://bugs.openjdk.org/browse/JDK-6245410): javax.swing.text.html.CSS.Attribute: BACKGROUND_POSITION is not w3c spec compliant

### Reviewers
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**) ⚠️ Review applies to [38c0f1a4](https://git.openjdk.org/jdk/pull/13114/files/38c0f1a470c9e78fe75d8af56ed1f48ad1f39ab9)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13114/head:pull/13114` \
`$ git checkout pull/13114`

Update a local copy of the PR: \
`$ git checkout pull/13114` \
`$ git pull https://git.openjdk.org/jdk.git pull/13114/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13114`

View PR using the GUI difftool: \
`$ git pr show -t 13114`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13114.diff">https://git.openjdk.org/jdk/pull/13114.diff</a>

</details>

### CODE REVIEW

**Review by aivanov-jdk — APPROVED** [review]

**Comment by aivanov-jdk** [comment]
What about `CSS.Attribute.TEXT_DECORATION`? I agree it should be handled separately, but I don't see a related bug linked in JBS to address the second issue from the bug description.

File: test/jdk/javax/swing/text/html/CSS/CSSAttributeComplianceTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+/*
+ * @test
+ * @bug 6245410
+ * @summary  Verifies CSS.Attribute: BACKGROUND_POSITION is w3c spec compliant
+ * @run main CSSAttributeComplianceTest
+ */
+
+import javax.swing.text.html.CSS.Attribute;
+
+public class CSSAttributeComplianceTest{
```
[the remark this answers is not in the document]
Comment (mrserb) [reply test/jdk/javax/swing/text/html/CSS/CSSAttributeComplianceTest.java:32]: Can we check that this default value will be parsed fine by some of our html component?

File: src/java.desktop/share/classes/javax/swing/text/html/CSS.java
Comment (kumarabhi006) [inline src/java.desktop/share/classes/javax/swing/text/html/CSS.java:1]: Please update the copyright year.

File: test/jdk/javax/swing/text/html/CSS/CSSAttributeComplianceTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (prsadhuk) [reply test/jdk/javax/swing/text/html/CSS/CSSAttributeComplianceTest.java]: Yes, we can extend the test in future, most probably for TEXT_DECORATION so I will keep the name same

The parsing expects the value to be in form of "% %" as can be seen here which also does the parsing and there's no regression observed in client tests

https://github.com/openjdk/jdk/blob/91f407d6fe285c44bcc25c1acdf5dc0c43be0172/src/java.desktop/share/classes/javax/swing/text/html/CSS.java#L2687-L2709

**Pull request closed without merging**